trunk-recorder/main.cc

```diff
@@ -539,7 +539,9 @@ bool load_config(string config_file) {
     config.log_dir = pt.get<std::string>("logDir", "logs");
     BOOST_LOG_TRIVIAL(info) << "Log Directory: " << config.log_dir;
     config.control_message_warn_rate = pt.get<int>("controlWarnRate", 10);
-    BOOST_LOG_TRIVIAL(info) << "Control channel warning rate: " << config.control_message_warn_rate;
+    BOOST_LOG_TRIVIAL(info) << "Control channel decoding warnings (< messages/second): " << config.control_message_warn_rate;
+    config.control_message_warn_updates = pt.get<float>("controlWarnUpdate", 3.0);
```

Comment:
In the property tree, you get `controlWarnUpdate` as a float, but don't you describe it as an int in seconds?